llvm/lib/Target/AMDGPU/SIMachineFunctionInfo.cpp

```diff
@@ -48,6 +48,9 @@ SIMachineFunctionInfo::SIMachineFunctionInfo(const Function &F,
   MaxNumWorkGroups = ST.getMaxNumWorkGroups(F);
   assert(MaxNumWorkGroups.size() == 3);
 
+  IsDynamicVGPREnabled =
+      ST.isDynamicVGPREnabled() || AMDGPU::getIsDynamicVGPR(F);
```

Comment:
Why isn't this an `&&`?

Reply:
Because the plan is to remove the subtarget feature after I update the internal users to rely on the attribute instead. In the meantime, both the attribute and the subtarget feature should work.

Reply:
Comment this